.github/workflows/job_build_windows.yml

```
@@ -254,7 +246,9 @@ jobs:
           artifacts: |
             ${{ env.BUILD_DIR }}/openvino_package.zip
             ${{ env.BUILD_DIR }}/openvino_tests.zip
-            ${{ env.INSTALL_WHEELS_DIR }}/wheels
             ${{ env.MANIFEST_PATH }}
+            ${{ env.STORE_WHEELS == 'true' && format('{0}/wheels', env.INSTALL_WHEELS_DIR) || '' }}
           storage_dir: ${{ env.PRODUCT_TYPE }}
           storage_root: ${{ env.ARTIFACTS_SHARE }}
+        env:
+          STORE_WHEELS: ${{ inputs.build-type != 'Debug' }}
```

I think it could be done w/o the additional env var.

```suggestion
            ${{ inputs.build-type != 'Debug' && format('{0}/wheels', env.INSTALL_WHEELS_DIR) || '' }}
          storage_dir: ${{ env.PRODUCT_TYPE }}
          storage_root: ${{ env.ARTIFACTS_SHARE }}
```

---

I'll rework this separately, we need to change that in several pipelines